### DISCUSSION TITLE
A help on the ZSH installation Wiki

### DISCUSSION THREAD

**Discussion by Heus-Sueh** [ohmyzsh/ohmyzsh#11512] [Ideas]
Recommend using the `sudo lchsh $USER` command on the Zsh installation Wiki if the user is at Fedora Silverblue or Kinoite 
![Captura de tela de 2023-02-21 22 08 24](https://user-images.githubusercontent.com/98482469/220494670-08c06850-bbee-4b74-b495-b3ad3a30e8c3.png)

https://github.com/ohmyzsh/ohmyzsh/wiki/Installing-ZSH

**Comment by carlosala** [comment]
If you can make a PR it'd be so good! https://github.com/ohmyzsh/wiki

**Reply by Heus-Sueh** [reply]
https://github.com/ohmyzsh/wiki/pull/21 :+1:
